### DISCUSSION TITLE
kata 10, modificar permisos de config.txt

### DISCUSSION THREAD

**Discussion by MauricioNovelo** [LaunchX-InnovaccionVirtual/CursoIntroPython#267] [Q&A]
Como se hace para que el config.py no tenga acceso a es archivo y se cumpla la condicional except Except

![image](https://user-images.githubusercontent.com/83683322/153991730-d342228c-1b6d-4bf6-8049-dc3a73c9ab38.png)

**Comment by doguedogue** [comment]
Genial WSL, puedes hacer varias cosas con permisos de lectura/escritura (chmod) o cambiar de dueño(chown) del archivo 👍🏼
